src/content/en/updates/2017/02/media-session.md

```diff
@@ -307,7 +307,7 @@ browser can't fetch them. Here's how you could implement this:
 
     const FALLBACK_ARTWORK_URL = 'fallbackArtwork.png';
     
-    self.addEventListener('install', event => {
+    addEventListener('install', event => {
       self.skipWaiting();
```

Comment:
As an editor, I'll take your word for it. As a student of web development, I'd like to know why?